scripts/install_third_party.sh

```diff
@@ -96,7 +96,9 @@ if [ ! "$NO_SKULPT" -a ! -d "$THIRD_PARTY_DIR/static/skulpt-0.10.0" ]; then
     # third party commands. These are only used for unit tests and generating
     # documentation and are not necessary when building Skulpt.
     sed -e "s/ret = test()/ret = 0/" $TOOLS_DIR/skulpt-0.10.0/skulpt/skulpt.py |\
-    sed -e "s/  doc()/  pass#doc()/" > $TMP_FILE
+    sed -e "s/  doc()/  pass#doc()/" |\
+    sed -e "s/ret = os.system(\"{0}/ret = 0 #os.system(\"{0}/" |\
```

Comment:
Could you file an issue for updating Skulpt (and for removing this stuff if possible)?

Reply:
Done: #6465.

Reply:
Also, it looks like upgrading Skulpt is not likely to resolve this, since a new version has not been released for quite a while, so maybe we don't need  #6465 after all.

Reply:
Closed.